README.md

```diff
@@ -15,3 +15,39 @@ the function and run automatically when you commit through a GitHub action.
 ## Runtime Analysis
 
 What is the worst-case big $\Theta$ time complexity of your algorithm?
+
+ok this is discusting here are the graphs i manually computed:
+![alt text](image.png)
+
+so i have four main steps in my process:
+1. Do the graphs have the same number of vertices and edges
+    -if not return early
+2. Insert every node into a priority queue weighted by the number of edges that verticie has.
+3. construct a tree based on the graph and the weighted nodes.
+    -the next node to be put into the tree will be based on the priority queue so we always construct our tree based on the most valuble node.
+4. compare the trees, if they have exactly the same structure then they are isomorphic.
```

Comment:
Are you using the definition of isomorphism we've used in class? This doesn't sound like you're trying to find the mapping that maps nodes in the two graphs to each other.